kibana_logger/Log_level.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
```

Comment:
please name the file "log_level.py"  the naming is a bit strange (upper case first letter and `_` )